.github/dependabot.yml

```diff
@@ -12,9 +13,12 @@ updates:
     groups:
       all:
         update-types:
+          - "minor"
```

Comment:
Ah, ok. I was not sure why we did not add `minor` there, also because it only creates a PR we still can reject, right?

Reply:
yes but then it will not open new one, only if a new version comes up, and then you will need to open manually which will be work that we try to eliminate using dependabot

Reply:
Ah, sure, sure, makes sense!